src/js/components/tasks/table.tsx

```diff
@@ -297,10 +322,15 @@ const TaskTable = ({
   };
   const taskDefaultSort = sortBy(tasks, [
     (item) => statusOrder[item.status],
-    'name',
+    (item) => item.name.toLowerCase(),
   ]);
   return (
-    <DataTable items={taskDefaultSort} id="epic-tasks-table" noRowHover>
+    <DataTable
+      items={taskDefaultSort}
+      id="epic-tasks-table"
+      className={viewEpicsColumn ? 'outdented_medium' : 'todo-remove-me'}
```

Comment:
@oluoluoxenfree  @jgerigmeyer can I please get some JS help on this? If I remove the `: 'todo-remove-me'` I think I get an error. Basically, I only need this class on the table when it is inside the tab panel.